.github/workflows/build.yml

```diff
@@ -279,6 +279,7 @@ jobs:
           cp ./jupytergis_qgis/dist/jupytergis*.whl jupytergis_qgis-0.0.1-py3-none-any.whl
           cp ./jupytergis_lite/dist/jupytergis*.whl jupytergis_lite-0.0.1-py3-none-any.whl
           cp ./jupytergis/dist/jupytergis*.whl jupytergis-0.0.1-py3-none-any.whl
+          pip install ./jupytergis_core/dist/jupytergis*.whl ./jupytergis_lab/dist/jupytergis*.whl ./jupytergis_lite/dist/jupytergis*.whl ./jupytergis/dist/jupytergis*.whl
```

Comment:
We're already handling the `whl`s in the jupyterlite-xeus environment here: https://github.com/geojupyter/jupytergis/blob/main/lite/environment.yml#L18

I believe we should fix https://github.com/jupyterlite/xeus/issues/227

Reply:
Adding a note that we need to keep it that way so that the Python API of JupyterGIS is available in the emscripten environment.

Installing those wheels in the host environment will result in an unneeded (harmless but unneeded) duplication of labextensions, so I'd suggest not applying this change.

Reply:
Thanks for opening this though! That made me realize the actual issue https://github.com/jupyterlite/xeus/issues/227 which was fairly hard to understand 😅